crates/types/src/pose_types.rs

```diff
@@ -0,0 +1,10 @@
+use serde::{Deserialize, Serialize};
+use serialize_hierarchy::SerializeHierarchy;
+
+#[derive(Debug, Clone, Serialize, Default, Deserialize, SerializeHierarchy)]
+pub enum PoseType {
+    OverheadArms,
+    ArmsBySide,
+    #[default]
+    NoPose,
```

Comment:
Why is this needed? Sounds like to be solved with `Option<PoseType>`

Reply:
Is now an `Option<PoseKind>`